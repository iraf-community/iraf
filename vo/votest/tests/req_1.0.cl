#
#  Demonstrate general use of VOTable documents.

# Set the test description string.
votest.descr = "Demonstrate general use of VOTable documents."


print ("------------------------------------------------------------------")
print ("Req  1:  Users shall be able to use a VOTable in places where")
print ("	 tasks accept tabular data in other formats (ASCII files,")
print ("	 FITS bintables, .tab files, etc) for input.")
print ("------------------------------------------------------------------")


#-------------------------------------------------------------------------

print ("Demonstration of the general use of VOTables is done using the")
print ("various individual 'votable' test scripts for access.")
print ("")
print ("")
