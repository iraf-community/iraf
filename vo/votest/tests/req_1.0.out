------------------------------------------------------------------
Req  1:  Users shall be able to use a VOTable in places where
	 tasks accept tabular data in other formats (ASCII files,
	 FITS bintables, .tab files, etc) for input.
------------------------------------------------------------------
Demonstration of the general use of VOTables is done using the
various individual 'votable' test scripts for access.


